### PR TITLE
fix: Auto update apk branch on each travis build

### LINKED ISSUE #94: Automatically build app in apk branch
Please use travis to automatically build the app in the apk branch.

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #94 

- Copy correct apk to build environment and upload

### CODE REVIEW

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -11,12 +11,28 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-
...
+mkdir $HOME/daily/
+cp -R /home/travis/build/fossasia/open-event-orga-app/app/build/outputs/apk/app-debug.apk $HOME/daily/
+cd $HOME
+
 git clone --quiet --branch=apk https://the-dagger:$GITHUB_API_KEY@github.com/fossasia/open-event-orga-app apk > /dev/null
-cp platforms/android/build/outputs/apk/android-debug.apk apk/test-android-debug.apk
 cd apk
+cp -Rf $HOME/daily/*  ./
+mv app-debug.apk sample-apk-${TRAVIS_BRANCH}.apk
+echo $TRAVIS_COMMIT > meta/deployment/commit_hash
```
Comment (niranjan94) [inline scripts/update-apk.sh]: These are not required for this. This is used only in open-event-android for deploying the generator to kubernetes.

File: scripts/update-apk.sh [hunk 2]
```diff
@@ -11,12 +11,28 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-
     exit 0
 fi
 
+mkdir $HOME/daily/
+cp -R /home/travis/build/fossasia/open-event-orga-app/app/build/outputs/apk/app-debug.apk $HOME/daily/
```
Comment (niranjan94) [inline scripts/update-apk.sh]: Why this process of creating a directory. Copying apk to it. Then copying to the branch and deleting the directory ? Can't you copy directly to the branch ?

File: scripts/update-apk.sh (hunk 1 above)
Comment (iamareebjamal) [inline scripts/update-apk.sh]: OK. Sorry, I just copied it from there. I'll change it

**Comment by iamareebjamal** [comment]
@niranjan94 Fixed

**Comment by iamareebjamal** [comment]
Travis build failed due to guava modified header, closing and reopening PR to trigger another build

**Comment by iamareebjamal** [comment]
@niranjan94 Travis is saying that git authentication failed. Can you please take a look?

**Comment by niranjan94** [comment]
@iamareebjamal will check that

**Comment by niranjan94** [comment]
@iamareebjamal token updated. Works now.

**Pull request merged**